cdap-ui/app/cdap/components/LogViewer/Playground.tsx

```diff
@@ -64,51 +67,23 @@ const LogViewerPlayground: React.FC<WithStyles<typeof styles>> = ({ classes }) =
       });
 
       setDataFetcher(dataFetch);
-
-      dataFetch.init().subscribe((logRes) => {
-        setLogs(logRes);
-      });
-    });
-  }
-
-  function previous() {
-    dataFetcher.getPrev().subscribe((res) => {
-      const newLogs = res.concat(logs);
-      setLogs(newLogs);
-    });
-  }
-
-  function next() {
-    dataFetcher.getNext().subscribe((res) => {
-      const newLogs = logs.concat(res);
-      setLogs(newLogs);
-    });
-  }
-
-  function getLast() {
-    dataFetcher.getLast().subscribe((res) => {
-      setLogs(res);
-    });
-  }
-
-  function getFirst() {
-    dataFetcher.getFirst().subscribe((res) => {
-      setLogs(res);
     });
   }
 
   return (
-    <div className={classes.root}>
+    <div>
       <h1>Log Viewer</h1>
 
-      <div className={classes.pipelineInfo}>
+      <div className={classes.padding}>
```

Comment:
I think we could name this into something more related to the log viewer like `pipelineInfo`, as we set it previously?

Reply:
This component will get deleted in subsequent PR. The playground is only there before the integration with real program that actually produces logs